cluster/juju/bundles/local.yaml

```diff
@@ -1,53 +1,18 @@
-kubernetes-local:
-  services:
-    kubernetes-master:
-      charm: local:trusty/kubernetes-master
-      annotations:
-        "gui-x": "600"
-        "gui-y": "0"
-      expose: true
-      options:
-        version: "local"
-    docker:
-      charm: cs:trusty/docker
-      num_units: 2
-      options:
-        latest: true
-      annotations:
-        "gui-x": "0"
-        "gui-y": "0"
-    flannel-docker:
-      charm: cs:~kubernetes/trusty/flannel-docker
-      annotations:
-        "gui-x": "0"
-        "gui-y": "300"
-    kubernetes:
-      charm: local:trusty/kubernetes
-      annotations:
-        "gui-x": "300"
-        "gui-y": "300"
-    etcd:
-      charm: cs:~kubernetes/trusty/etcd
-      annotations:
-        "gui-x": "300"
-        "gui-y": "0"
-  relations:
-    - - "flannel-docker:network"
-      - "docker:network"
-    - - "flannel-docker:network"
-      - "kubernetes-master:network"
-    - - "flannel-docker:docker-host"
-      - "docker:juju-info"
-    - - "flannel-docker:docker-host"
-      - "kubernetes-master:juju-info"
-    - - "flannel-docker:db"
-      - "etcd:client"
-    - - "kubernetes:docker-host"
-      - "docker:juju-info"
-    - - "etcd:client"
-      - "kubernetes:etcd"
-    - - "etcd:client"
-      - "kubernetes-master:etcd"
-    - - "kubernetes-master:minions-api"
-      - "kubernetes:api"
-  series: trusty
+services:
+  kubernetes:
+    charm: local:trusty/kubernetes
+    annotations:
+      "gui-x": "600"
+      "gui-y": "0"
+    expose: true
+    num_units: 2
+  etcd:
+    charm: cs:~lazypower/trusty/etcd
```

Comment:
I'll bring this in on the next update w/ the docs. @zmerlynn - is that still going to be pointed here or have the docs migrated to their new home already?